src/home_components/battery.py

```diff
@@ -676,3 +680,9 @@ def check_feasible_bat(self, prm, passive_ext):
             time += 1
 
         return feasible
+
+    def active_reactive_power_car(self):
```

Comment:
have loss_ch/charge/discharge be an input to the method OR make sure that you have updated the method attributes before calling the method